components/style/stylist.rs

```diff
@@ -1303,11 +1272,21 @@ impl Stylist {
         }
 
         // Step 3b: XBL rules.
-        let cut_off_inheritance =
-            element.get_declarations_from_xbl_bindings(
-                pseudo_element,
-                applicable_declarations,
-            );
+        let cut_off_inheritance = element.each_xbl_stylist(|stylist| {
+            // nsXBLPrototypeResources::LoadResources() loads Chrome XBL style
```

Comment:
While you're here, please `s/nsXBLPrototypeResources::LoadResources()/ ServoStyleSet::CreateXBLServoStyleSet()/`, and delete "Chrome" in the comment because author XBL stylesheets also added under eAuthorSheetFeatures level.

Reply:
Done, thanks for the review! :)